core/src/sumeragi/main_loop.rs

```diff
@@ -433,31 +422,31 @@ impl Sumeragi {
         #[allow(clippy::suspicious_operation_groupings)]
         match (message, role) {
             (BlockMessage::BlockSyncUpdate(BlockSyncUpdate { block }), _) => {
-                let block_hash = block.hash();
-                info!(%addr, %role, hash=%block_hash, "Block sync update received");
+                let block_height = block.header().height;
```

Comment:
I think height and hash is important to report, there could be one the one block candidate for any given height.